python/podio_class_generator.py

```diff
@@ -305,46 +354,52 @@ def _process_datatype(self, name, definition):
     """Process one datatype"""
     datatype = self._preprocess_datatype(name, definition)
 
-    # ROOT schema evolution preparation
-    # Compute and prepare the potential schema evolution parts
-    schema_evolution_datatype = copy.deepcopy(datatype)
-    needs_schema_evolution = False
-    for member in schema_evolution_datatype['Members']:
-      if member.is_array:
-        if member.array_type in self.root_schema_dict:
-          needs_schema_evolution = True
-          self._replace_component_in_paths(member.array_type, member.array_type + self.old_schema_version,
-                                           schema_evolution_datatype['includes_data'])
-          member.full_type = member.full_type.replace(member.array_type, member.array_type + self.old_schema_version)
-          member.array_type = member.array_type + self.old_schema_version
+    if self.proglang == "julia":
+      self._fill_templates('MutableStruct', datatype)
+
```

Comment:
```suggestion
    return datatype
```
Can we return early here and not have a huge diff below just from re-indenting everything from the additional `if self.proglang == "cpp":`?